share/spice/quote_of_the_day/quote_of_the_day.css

```diff
@@ -0,0 +1,57 @@
+.quote__of__the__day--container {
+    overflow: hidden;
+    margin-left: -2.8em;
+}
+
+.quote__of__the__day {
+    quotes: "“" "”";
+    font-size: 1.5em;
+    font-weight: 300;
+}
+
+.zci--quote_of_the_day .zci__body {
+    margin-left: -2em;
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    padding-left: 2.8em;
+}
+
+.quote__of__the__day:before {
+    color: #D4D4D4;
+    content: open-quote;
+    float: left;
+    font-size: 4em;
+    margin-top: -0.25em;
+    margin-right: 0.1em;
+}
+
+.quote__attribution {
+    color: #999;
+    float: left;
+    margin-top: 5px;
+}
+
+.zci--quote_of_the_day .zci__content.chomp {
+    max-height: none;
+    margin-bottom: 0;
+}
+
+.zci--quote_of_the_day .zci__body.zci__body--info {
+    padding-left: 0.25em;
+}
+
+.is-mobile .quote__of__the__day--container {
+    margin-left: 0px;
+    margin-bottom: 0.5em;
+}
+
+.is-mobile .zci--quote_of_the_day .zci__more-at {
+    margin-left: 40px;
+
+}
+
+.is-mobile .zci--quote_of_the_day .brainy__attribution {
```

Comment:
Got some `.brainy` here. I suggest you do a quick `grep` for `brainy` to check if there is anything else that needs updating.